packages/kotti-ui/source/constants.ts

```diff
@@ -1,3 +1,4 @@
+export const ISO8601_DAY = 'YYYY-MM-DD' as const
```

Comment:
does this not exist already as ISO8601? should exist because we have a const for the KtFieldDate format

Reply:
Yes but the format that `element-ui` uses is different and not compatible with the one dayjs uses.
<img width="513" alt="Screenshot 2023-11-07 at 12 31 49" src="https://github.com/3YOURMIND/kotti/assets/744091/b713c400-d16c-48aa-b210-2f5aac72394b">
<img width="428" alt="Screenshot 2023-11-07 at 12 39 37" src="https://github.com/3YOURMIND/kotti/assets/744091/abb74e28-15d9-42ee-9cfe-e0c96024c978">

Reply:
Interesting, got it, then I'd just call it `ISO8601`, there's no point in adding _DAY

we add _SECONDS for the other const because it's the full format